app/src/main/java/org/kinecosystem/kinit/network/OnboardingService.kt

```diff
@@ -121,26 +120,22 @@ class OnboardingService(context: Context, private val appLaunchApi: OnboardingAp
         deviceUtils.timeZoneDebugging(analytics)
         val displayMetrics = applicationContext.resources.displayMetrics
         val call = appLaunchApi.register(OnboardingApi.RegistrationInfo(userId,
-            deviceModel = deviceUtils.deviceName(),
-            timeZone = timeZoneWithoutGMT,
-            deviceId = deviceUtils.deviceId(),
-            appVersion = appVersion,
-            screenWidth = displayMetrics.widthPixels,
-            screenHeight = displayMetrics.heightPixels,
-            density = displayMetrics.densityDpi))
+                deviceModel = deviceUtils.deviceName(),
+                timeZone = timeZoneWithoutGMT,
+                deviceId = deviceUtils.deviceId(),
+                appVersion = appVersion,
+                screenWidth = displayMetrics.widthPixels,
+                screenHeight = displayMetrics.heightPixels,
+                density = displayMetrics.densityDpi))
         call.enqueue(object : Callback<StatusResponse> {
             override fun onResponse(call: Call<StatusResponse>?,
                                     response: Response<StatusResponse>?) {
 
                 if (response != null && response.isSuccessful) {
                     updateConfig(response)
                     updateToken()
-                    analytics.logEvent(UserRegistered())
+                    analytics.logEvent(Events.Business.UserRegistered())
                     userRepo.isRegistered = true
-                    wallet.initKinWallet()
-                    if (!userRepo.isPhoneVerificationEnabled) {
```

Comment:
check if firebase allows emulator with test numbers to run - if so remove checks